src/module/skb/bpf/skb_hook.bpf.c

```diff
@@ -125,7 +173,7 @@ static __always_inline int process_packet(struct retis_raw_event *event,
 	 * the eth header.
 	 */
 	} else if (is_network_data_valid(skb)) {
-		u16 etype = BPF_CORE_READ(skb, protocol);
+		u16 etype = skb_protocol(skb);
```

Comment:
While looking at this I thought: would it be desirable to flag the eth header info as "fake" in this case so we can represent it as ???? -> ??? instead of all zeros? for someone not knowing retis internals it might be confusing to see those headers and people might thing there is something wrong with the e.g: driver.

Reply:
Makes sense. Instead of reporting `???` we can just avoid filling the eth information in the event. I'll add a commit to this PR.